.gitignore

```diff
@@ -30,3 +30,4 @@ coverage/
 .tool-versions
 .rspec_status
 .rdbg_history
+.idea/
```

Comment:
Can you add a comment of what this is intended to ignore? Something like

```
# ignore Jetbrains IntelliJ project-config directory
.idea/
```

That way folks who don't use IDEA have some context w/o having to run `git blame` on it

Reply:
Thanks. I updated it